java/src/main/java/com/powsybl/dataframe/network/adders/ShuntDataframeAdder.java

```diff
@@ -41,7 +45,10 @@ public class ShuntDataframeAdder implements NetworkElementAdder {
             SeriesMetadata.ints("section_count"),
             SeriesMetadata.doubles("target_deadband"),
             SeriesMetadata.doubles("target_v"),
-            SeriesMetadata.strings("model_type")
+            SeriesMetadata.strings("model_type"),
+            SeriesMetadata.strings("busbar_section_id"),
```

Comment:
Note for later: it's not ideal to have the new columns in the metadata of existing dataframes, but let's keep it like that for now.